ts/src/pro/binance.ts

```diff
@@ -1232,6 +1232,10 @@ export default class binance extends binanceRest {
         if (firstMarket['contract']) {
             type = firstMarket['linear'] ? 'future' : 'delivery';
         }
+        const isSpot = (type === 'spot');
+        let timezone = undefined;
+        [ timezone, params ] = this.handleParamString2 (params, 'timezone', 'timeZone', undefined);
```

Comment:
again we should add this to the docs, otherwise users. can't see it

Reply:
also using only `timezone` is enough I think, having the `timeZone` variation is not needed

Reply:
two things here:
- `timeZone` is exchange specific parameter name for `REST` endpoint, so it thought it to support here too, but right, it's odd here
- should we already add that in docs, because it's not much unified atm and only binance-specific? oky, i'll add

Reply:
pushed